monthly-blog.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+usage () {
+echo "Usage: monthly-blog.sh YYYY M path/to/mathlib"
+echo ""
+echo "Where YYYY is a 4-digit year, and M is the number of the month."
+echo "Example: monthly-blog.sh 2021 8 ~/mathlib"
+echo "It is important that M is a natural number between 1 and 12."
+echo "Do not write 08 to denote August. Write 8 instead."
+echo ""
+echo "Important: make sure that the mathlib clone is clean,"
+echo "and points to an up-to-date copy of master."
+}
+
+year=$1
+month=$2
+mathlib=$3
+
```

Comment:
Don't we need some code here calling `usage` if there are missing arguments?

Reply:
Sure, but I'm not a good bash hacker. So I mostly wrote a template that could get other started.